Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.5
```

Comment:
Seems to be there. https://hub.docker.com/layers/library/elixir/1.5/images/sha256-d7e3150739b34ed2cd761580877b5947ea7d13232626c19d88aaeca3e8b4f2ed. I don't think we'll downgrade. If anything, we should upgrade. Thanks for the PR.